src/sentry/static/sentry/app/components/buttons/button.jsx

```diff
@@ -0,0 +1,107 @@
+import React, {PropTypes} from 'react';
+import {Link} from 'react-router';
+import classNames from 'classnames';
+
+import '../../../less/components/button.less';
+
+const Button = React.createClass({
```

Comment:
nit: to me it feels like it should be 'size' and an enum vs props that can conflict with each other

Reply:
@billyvg sorry I missed these comments before. I think the boolean stuff works here, but I wonder if we'll run into issues with the approach on other components. I'm 👍 for whatever approach we can consistently implement everywhere.

Reply:
np, I changed it back because it's easier for tooling to catch mistakes.